bk-api/test_bk_api.py

```diff
@@ -1,55 +1,88 @@
 import bk_api
 import datetime
-import tempfile
 import pytest
-import os
-
 import json
 import io
-import time
+
 
 # from https://flask.palletsprojects.com/en/1.1.x/testing/
 @pytest.fixture
 def client():
+    # TODO(sean) It would be nice to setup a fresh database here, so different unit tests can't affect each other.
+    # For example, we could generate some BeekeeperTestRandomID database and init the tables.
     app = bk_api.app
-    db_fd, app.config['DATABASE'] = tempfile.mkstemp()
```

Comment:
These don't actually do anything and were copied from Flask's testing Flask apps" document.